docs/src/guide/installation.md

```diff
@@ -0,0 +1,43 @@
+# Installation
+
+DFTK is not yet registered in the [General](https://github.com/JuliaRegistries/General)
+registry of Julia.
+Instead you can obtain it from
+the [MolSim](https://github.com/JuliaMolSim/MolSim.git) registry,
+which contains a bunch of packages related to performing molecular simulations in Julia.
+At least **Julia 1.3** is required.
+
+First add `MolSim` to your installed registries. For this use
+```
+] registry add https://github.com/JuliaMolSim/MolSim.git
+```
+from a Julia command line.
+Afterwards you can install DFTK like any other package in Julia:
+```
+] add DFTK
+```
+or if you like the bleeding edge:
+```
+] add DFTK#master
+```
```

Comment:
Bleeding edge makes it sound dangerous, but we're pretty careful with what we do on master. I'd rather tell users to install master by default

Reply:
Actually once we go to 0.1.0 I plan to release much more often, so this will not really be an issue either.